resource-sharing-partners/src/test/java/no/sikt/rsp/ResourceSharingPartnerTest.java

```diff
@@ -128,6 +128,16 @@ public void shouldBeAbleToReadAndPostRecordToAlma() throws IOException {
         assertThat(response, is(1));
     }
 
+    @Test
+    public void shouldBeAbleToReadAndPutRecordToAlma() throws IOException {
+        var baseBibliotek0030100 = IoUtils.stringFromResources(Path.of(BASEBIBLIOTEK_0030100_XML));
+        var uri = s3Driver.insertFile(randomS3Path(), baseBibliotek0030100);
+        var s3Event = createS3Event(uri);
+        Integer response = resourceSharingPartnerHandler.handleRequest(s3Event, CONTEXT);
+        assertThat(response, is(notNullValue()));
+        assertThat(response, is(1));
```

Comment:
Vet ikke om det er nødvendig, men vi kan også verifisere at det ble gjort kall mot wiremocken (lært av Ketil)

Reply:
verify it is!